src/jobs/gradle_owasp_dependency_check.yml

```diff
@@ -57,32 +57,4 @@ steps:
   - slack/notify:
       event: fail
       channel: << parameters.slack_channel >>
-      custom: |
-        {
-          "blocks": [
-            {
-              "type": "context",
-              "elements": [
-                {
-                  "type": "mrkdwn",
-                  "text": ":circleci-${CCI_STATUS}: CircleCI job *${CIRCLE_JOB}* ${CCI_STATUS}"
-                }
-              ]
-            },
-            {
-              "type": "section",
-              "text": {
-                "type": "mrkdwn",
-                "text": "*${CIRCLE_PROJECT_REPONAME}* failed ${CIRCLE_JOB}"
-              },
-              "accessory": {
-                "type": "button",
-                "text": {
-                  "type": "plain_text",
-                  "text": "View job"
-                },
-                "url": "${CIRCLE_BUILD_URL}"
-              }
-            }
-          ]
-        }
+      custom: <<include(files/slack_failed.tpl.json)>>
```

Comment:
Decided to create `slack_failed.tpl` rather than use the existing `slack_failed_with_results.tpl` because that one has a second `context` section which prints _No output, view job for details._ if it cannot parse the output. Wasn't sure about changing the Slack message for the existing jobs, but happy to if folks think don't think it's a big issue